app/models/redis/site_setting.rb

```diff
@@ -0,0 +1,19 @@
+class Redis::SiteSetting
+  PREFIX = "site_setting"
+
+  class << self
```

Comment:
Prefer `def self.bool` style for defining static methods for consistency with the rest of metasmoke

Reply:
Ohhhh, I thought you were commenting on line 2, not line 4. I really prefer the class << self notation, but I can change it.